api/grpcserver/http_server.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"github.com/rs/cors"
```

Comment:
The errors listed here are because you need to add the dependency to `go.mod`. Run `go mod tidy` and commit the changes to fix it 🙂

Reply:
👍